src/erldns_query_throttle.erl

```diff
@@ -103,6 +106,7 @@ code_change(_PreviousVersion, State, _Extra) ->
 
 
 % Internal API
+-spec(maybe_throttle(inet:ip_address() | inet:hostname()) -> throttle_result()).
```

Comment:
Yeah. Y'all already defined it in master: `https://github.com/aetrion/erl-dns/blob/master/src/erldns_query_throttle.erl#L39`